pkg/operators/ustack/ustack.go

```diff
@@ -247,6 +254,46 @@ func (o *OperatorInstance) init(gadgetCtx operators.GadgetContext) error {
 					if addressesStr != "" {
 						addressesField.PutString(data, addressesStr)
 					}
+
+					if o.buildIDMap != nil && o.buildIDMap() != nil {
+						// struct bpf_stack_build_id is part of Linux UAPI:
+						// https://github.com/torvalds/linux/blob/v6.14/include/uapi/linux/bpf.h#L1451
+						type bpfStackBuildID struct {
+							status     int32
+							buildID    [unix.BPF_BUILD_ID_SIZE]uint8
+							offsetOrIP uint64 // Union of offset and ip
+						}
+						buildIDMapValueSize := o.buildIDMap().ValueSize()
+						buildIDBuf := [ebpftypes.UserPerfMaxStackDepth * unsafe.Sizeof(bpfStackBuildID{})]byte{}
+						if int(buildIDMapValueSize) != len(buildIDBuf) {
+							logger.Warnf("build id map has unexpected value size %d instead of %d", buildIDMapValueSize, len(buildIDBuf))
+							return nil
+						}
```

Comment:
This is changed on the "ustack operator: add client-side support (gadgetctl, kubectl-gadget)" commit. Why not to squash "ustack: add buildid field for client-side symbolization", "symbolizer: move common functions to new file" & "ustack operator: add client-side support (gadgetctl, kubectl-gadget)"?